pkg/util/namespace_filter.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2018 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"github.com/golang/glog"
+	apiv1 "k8s.io/api/core/v1"
+	"regexp"
+	"strings"
+)
+
+type NamespaceFilterConfig struct {
+	Namespace       string
+	NamespaceFilter string
+}
+
+func isNamespaceFilterEnabled(namespaceFilterConfig *NamespaceFilterConfig) bool {
+	enableFilter := false
+	if namespaceFilterConfig == nil {
+		glog.Info("No filter on namespaces is applied")
+		return enableFilter
+	}
+	if namespaceFilterConfig.Namespace != apiv1.NamespaceAll {
+		glog.Info("A namespace is specified. namespace-filter will be ignored even if it is specified")
```

Comment:
missing check non empty namespaceFilterConfig.Namespace (I know you already have a default value "All", but this function dont know it)

Reply:
Done